src/vm/ceeload.cpp

```diff
@@ -6461,6 +6461,36 @@ void Module::NotifyDebuggerUnload(AppDomain *pDomain)
 }
 
 #if !defined(CROSSGEN_COMPILE)
+using GetTokenForVTableEntry_t = mdToken(STDMETHODCALLTYPE*)(HMODULE module, BYTE**ppVTEntry);
```

Comment:
Why not just `typedef?

Or is this the new preferred way to say `typedef` in C++ 11 ?

Reply:
This is the new way to say `typedef` in C++11. The difference isn't too big, except for function types, where it gets a lot cleaner.